react_ui/src/components/map_data_points/MapDataPointModal.tsx

```diff
@@ -79,6 +79,8 @@ class MapDataPointModal extends React.Component<
     const credit = `${
       note.created_by && (note.created_by as CreatedBy).username
         ? (note.created_by as CreatedBy).username
+        : user
+        ? user.username
```

Comment:
Could we make this a bit easier to read? Or at least use optional chaining (with ?)